variables.tf

```diff
@@ -271,6 +271,21 @@ variable "log_retention_in_days" {
   default     = 30
 }
 
+variable "maintenance_window" {
+  type = object({
+    allowed = optional(list(object({
```

Comment:
I dont understand this failure in the CI:
https://github.com/Azure/terraform-azurerm-aks/actions/runs/3134168836/jobs/5088466915#step:10:37

Is the `prepr-check` failing because of `optional` ? I understand now that this is supported in Terraform 1.3.

I see that we install terraform here:
https://github.com/Azure/terraform-azurerm-aks/blob/master/.github/workflows/pr-check.yaml#L16

this should install always the latest Terraform version ? How do we check which exact Terraform version we are using in the CI ?

thanks